### DISCUSSION TITLE
Love your idea about this project

### DISCUSSION THREAD

**Discussion by mashuk999** [wanghaisheng/tiktoka-studio-uploader#13] [Ideas]
Even i was searching for similar project where we can upload the video in a headless method. I store my videos on my server for some automated processing but due to x11 is not available i have to download those in my rig to upload them. With this project I will be able to directly upload them. Awesome.

Heard you are currently working on some tasks, if possible can you make them on Github projects or trello. I will be really happy to help you on some task :)

**Comment by wanghaisheng** [comment]
during these days just make it gui available to help others non coding guys .
if any of my repo got your interest you can make contribution.
share this repo with friends over youtube  tiktok is better
sponsorship is even better
